hivemq-edge/src/frontend/src/extensions/datahub/components/helpers/ReactFlowSchemaForm.tsx

```diff
@@ -11,12 +11,28 @@ import {
   getUiOptions,
   ErrorListProps,
   TranslatableString,
+  TitleFieldProps,
 } from '@rjsf/utils'
 import { GenericObjectType } from '@rjsf/utils/src/types.ts'
 import validator from '@rjsf/validator-ajv8'
-import { Alert, AlertTitle, Box, FormControl, List, ListIcon, ListItem, Text } from '@chakra-ui/react'
+import { Alert, AlertTitle, Box, Divider, FormControl, Heading, List, ListIcon, ListItem, Text } from '@chakra-ui/react'
 import { WarningIcon } from '@chakra-ui/icons'
 
+// overriding the heading definition
```

Comment:
question(non-blocking): do we actually need the comment?

Because to me personally it's very clear what the code is doing.

```suggestion
```

Reply:
Ah, that's a good one. 
Technically no. 
But then I'm replicating the original piece of code, just to change two characters. But I don't have the history of the changes to see the difference. So I keep track of the reason for the override. 
But then, I got lazy! I used the shorthand for the object construction, reusing the same name:
```
      templates={{
        DescriptionFieldTemplate,
        FieldTemplate,
        ErrorListTemplate,
        TitleFieldTemplate,
      }}
```
So, the best way should be to rename my template in a meaningful way, e.g. `TitleFieldTemplateWithH2` and get no too lazy:

```
      templates={{
        DescriptionFieldTemplate,
        FieldTemplate,
        ErrorListTemplate,
        TitleFieldTemplate: TitleFieldTemplateWithH2,
      }}
```

Reply:
I got a ticket for rationalising the templates across both `Edge`, `DataHub` and potentially an external module. Will get a naming/commenting scheme in place